src/js/tabs/osd.js

```diff
@@ -1181,6 +1181,21 @@ OSD.loadDisplayFields = function() {
             positionable: true,
             preview: "#9876",
         },
+        UP_DOWN_REFERENCE: {
+            name: 'UP_DOWN_REFERENCE',
+            text: 'osdTextElementUpDownReference',
+            desc: 'osdDescUpDownReference',
+            defaultPosition() {
+                let position = 238;
+                if (OSD.constants.VIDEO_TYPES[OSD.data.video_system] !== 'NTSC') {
+                    position += FONT.constants.SIZES.LINE;
+                }
+                return position;
+            },
+            draw_order: 465,
+            positionable: true,
+            preview: 'U',
```

Comment:
This will also need to include the effective area that this element can span / interfere with, see `ARTIFICIAL_HORIZON` for how to do this.

Reply:
@bobbycisneros: This is still open, and required to allow users to position this without clipping the edges. This should match the screen area that the firmware constrains this element to.